app/src/main/java/co/nimblehq/compose/ecommerce/model/Category.kt

```diff
@@ -0,0 +1,45 @@
+package co.nimblehq.compose.ecommerce.model
+
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Immutable
+import co.nimblehq.compose.ecommerce.R
+
+@Immutable
+data class Category(
+    val id: Long,
+    val name: String,
+    @DrawableRes val image: Int
+)
+
+val categories = listOf(
```

Comment:
Do you have a plan to remove this so far, Is it just mock data or the real data🙏

Reply:
@Wadeewee We will use it for mocking the data. We don't have the API to do integration. I will create a fake repository to contain all the data. How do you think?

Reply:
it's ok just mock the data for now and make it more focused on the UI, @hoangnguyen92dn. Having more layers will lose the focus of the developers when reading the codebase so I advise we keep mocking it directly in the Model classes.

Reply:
I agree to keep mock data in model class but would it be good if we leave TODO or some comment why we have to use    mock data for now and we can integrate if we have an api later something like this What do you think? @hoangnguyen92dn